components/GalleryCard/GalleryCard.module.scss

```diff
@@ -11,7 +11,7 @@
 .thumbnail {
     @include border-radius-s;
 
-    height: 340px !important;
+    height: 236px !important;
```

Comment:
Shouldn't it use the same aspect-ratio mixin as the story cards? I guess the idea was to make all card images have the same ratio?

Reply:
Yeah will update, will have same as the medium story card